src/main/java/io/jenkins/plugins/sprp/models/Credential.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.plugins.sprp.models;
+
+public class Credential {
+    private String credentialId;
+    private String variable;
```

Comment:
Maybe it makes sense to reconsider the name. in some cases it is just a variable name prefix

Reply:
sure :+1:

Reply:
One thing we can do is : 
```
environment:
  variable_1: value_1
  variable_2: value_2

  credentials:
    - credentialId : fileCredentialId
      variable : FILE
    - credentialId : dummyGitRepo
      variable : LOGIN
```